### PR TITLE
HITL fan test

### PR DESCRIPTION
**Pull request by robbederks**
Test fan controller on embedded pandas

### CODE REVIEW

File: tests/hitl/conftest.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class PandaGroup:
   GEN2 = (Panda.HW_TYPE_BLACK_PANDA, Panda.HW_TYPE_UNO, Panda.HW_TYPE_DOS) + H7
   GPS = (Panda.HW_TYPE_BLACK_PANDA, Panda.HW_TYPE_UNO)
   GMLAN = (Panda.HW_TYPE_WHITE_PANDA, Panda.HW_TYPE_GREY_PANDA)
+  EMBEDDED = (Panda.HW_TYPE_DOS, Panda.HW_TYPE_TRES)
```
Comment (adeebshihadeh) [inline tests/hitl/conftest.py:27]: we refer to this as "internal" in the python lib

File: tests/hitl/7_embedded.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+from panda.tests.hitl.conftest import PandaGroup
+
+@pytest.mark.test_panda_types(PandaGroup.EMBEDDED)
+def test_fan_controller(p):
+  for power, expected_rpm in [(50, 3250), (100, 6500)]:
+    p.set_fan_power(power)
+    time.sleep(10)
+    assert 0.95 * expected_rpm <= p.get_fan_rpm() <= 1.05 * expected_rpm
+  p.set_fan_power(0)
```
Comment (adeebshihadeh) [inline tests/hitl/7_embedded.py:12]: no cleanup needed, it's generically handled by the fixture

File: tests/hitl/7_embedded.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+@pytest.mark.test_panda_types(PandaGroup.EMBEDDED)
+def test_fan_cooldown(p):
+  # if the fan cooldown doesn't work, we get high frequency noise on the tach line 
+  # while the rotor spins down. this makes sure it never goes beyond the expected max RPM
+  p.set_fan_power(100)
+  time.sleep(3)
+  p.set_fan_power(0)
+  for _ in range(5):
+    assert p.get_fan_rpm() <= 7000
```
Comment (adeebshihadeh) [inline tests/hitl/7_embedded.py:22]: add another assert on the tach irq fault status?

File: tests/hitl/7_embedded.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import time
+import pytest
+
+from panda.tests.hitl.conftest import PandaGroup
+
+@pytest.mark.test_panda_types(PandaGroup.EMBEDDED)
+def test_fan_controller(p):
+  for power, expected_rpm in [(50, 3250), (100, 6500)]:
```
Comment (adeebshihadeh) [inline tests/hitl/7_embedded.py:8]: expected rpm should be pulled out into a function, perhaps in the Panda class

File: tests/hitl/7_embedded.py (hunk 3 above)
Comment (robbederks) [reply tests/hitl/7_embedded.py:22]: faults are checked by the fixture as well!

**Comment by robbederks** [comment]
This can be merged once we're ready to merge #1335, which fixes the failing test cases.

**Pull request merged**